deployment/helm/balloons/templates/daemonset.yaml

```
@@ -91,11 +94,15 @@ spec:
         hostPath:
           path: /var/run/nri
           type: DirectoryOrCreate
-      {{- if .Values.nri.patchContainerdConfig }}
+      {{- if .Values.nri.patchRuntimeConfig }}
       - name: containerd-config
         hostPath:
           path: /etc/containerd/config.toml
-          type: File
+          type: FileOrCreate
```

After chatting with @fmuyassarov:

```suggestion
          path: /etc/containerd/
          type: DirectoryOrCreate
```

WDYT @klihub @kad @askervin? Now this would be safer but create empty config dir for the "other" runtime

---

Creating an empty directory on host doesn't sound a major sin to me. Creating an empty file would be worse as it might conflict with packages. And in my opinion mounting full /etc would be worse in security point of view. So this would be the best solution I can think of.

---

> Creating an empty directory on host doesn't sound a major sin to me.

My thinking, too

---

Agreed, fixed.

---

I think you're overcomplicating that. Just mount whole /etc. that directory must exist. Mounting it is also a common pattern, e.g. in CNI installation manifests.